R/json.tabular.to.data.frame.R

```diff
@@ -11,12 +11,13 @@ NULL
 # This is a hack to work around a bug in jsonlite where null's are not
 # converted to NA when simplifyVector=FALSE
 .null.to.na <- function(l) {
-  rs.class <- data.class(l)
-  return(switch(rs.class,
-    "NULL"=NA,
-    "list"=lapply(l, .null.to.na),
+  if (is.null(l)) {
```

Comment:
Huh, I'm surprised it makes this much difference. But I guess profiling is _supposed to_ surprise you :)